config/confighttp/confighttp.go

```diff
@@ -74,6 +74,10 @@ type HTTPClientSettings struct {
 	// IdleConnTimeout is the maximum amount of time a connection will remain open before closing itself.
 	// There's an already set value, and we want to override it only if an explicit value provided
 	IdleConnTimeout *time.Duration `mapstructure:"idle_conn_timeout"`
+
+	// DisableKeepAlives, if true, disables HTTP keep-alives and will only use the connection to the server
+	// for a single HTTP request.
+	DisableKeepAlives *bool `mapstructure:"disable_keep_alives"`
```

Comment:
Why are you using a boolean pointer? Can a boolean do the job?

Reply:
No special reason, boolean is also ok.